_field-types/supported-field-types/knn-vector.md

```diff
@@ -47,6 +47,28 @@ PUT test-index
 ```
 {% include copy-curl.html %}
 
+## Vector workload modes
```

Comment:
can we have table of mode, compression and which engine will be used in the docs?